.github/workflows/auto-approve-renovate.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto-Approve Renovate
```

Comment:
Were we able to test this out?

Reply:
Didn't test it, just checked around to make sure `renovate[bot]` was the proper author format. I figured it would be fine to check on come Monday, but I'm happy to merge it to a local fork with a custom schedule if we want to prove it out first.